front/src/createBuildingMesh.js

```diff
@@ -13,8 +13,25 @@ function zFrom8bitsInt(z, MIN_Z, MAX_Z){
     return z*(MAX_Z - MIN_Z)/(((1 << 8)-1)) + MIN_Z;
 }
 
+var intToMeshType = {
```

Comment:
Move this to their own file so it can be shared with 3dsFormatToAntsBinaryBuffer.js
Right now, 3dsFormatToAntsBinaryBuffer.js hardcodes the values (with associated comments) which isn't good as it may fall out of sync. If both share the mapping, no risk to fall out of sync.